.github/workflows/main.yaml

```diff
@@ -0,0 +1,56 @@
+name: CI
+
+on:
+  # runs on 12:55 (5:55AM PST) & 23:55 UTC
+  schedule:
+    - cron:  '55 12,23 * * *'  
```

Comment:
Just to be 100% safe and ensure we're running each 12 hours a part, we should either run the first flow at 11:55 UTC or the second flow at 00:55 UTC